src/components/ModelSteps/Contact.jsx

```diff
@@ -135,14 +137,35 @@ const Contact = () => {
 
 const ContactWrapper = ({ currentStep }) => {
   const { t } = useI18n()
+  const client = useClient()
   const { illustration, text } = currentStep
-  const { formSubmit } = useFormData()
+  const { formSubmit, formData } = useFormData()
   const [onLoad, setOnLoad] = useState(false)
+  const [confirmReplaceFileModal, setConfirmReplaceFileModal] = useState(false)
+
+  const cozyFiles = formData.data.filter(d => d.file.constructor === Blob)
 
-  const submit = () => {
+  const closeConfirmReplaceFileModal = () => setConfirmReplaceFileModal(false)
+  const openConfirmReplaceFileModal = () => setConfirmReplaceFileModal(true)
+
+  const submit = useCallback(() => {
     setOnLoad(true)
     formSubmit()
-  }
+  }, [formSubmit])
+
+  const onClickReplace = useCallback(
+    isReplace => {
+      ;(async () => {
```

Comment:
```suggestion
      (async () => {
```

Reply:
`eslint` imposes this semicolon in this case, I prefer this than using a named function or changing this rule